knox_test.go

```diff
@@ -28,9 +30,9 @@ type requestCredentialTest struct {
 func TestRequestCredential(t *testing.T) {
 	cred_type := "BankCard"
 	mock_controller := gomock.NewController(t)
-	mock_wallet := knox_mock.NewMockWallet(mock_controller)
+	mock_wallet := s_mock.NewMockDynamicSigner(mock_controller)
```

Comment:
variable and package name mock_wallet ans s_mock are underscored, we agreed on camelCase

Reply:
The naming conventions for packages are either camel case or underscore - https://github.com/knox-networks/gomono

Reply:
I think its better to have package names be lowercase (and underscore if necessary). I don't think camel case works for packages